content/pages/healthcare/apply.md

```diff
@@ -66,7 +66,6 @@ To update your personal, insurance, or financial information from a previous app
 ### Additional forms for caregivers, National Guard, and Reserve Servicemembers
```

Comment:
Sure. If you want removal without replacing how we're addressing National Guard needs, then let's not overthink the copy. @alexose I think you can just straight up remove the National Guard references and relevant commas.

ARE YOU A CAREGIVER ~, NATIONAL GUARD,~ OR RESERVE SERVICEMEMBER? THIS FORM’S NOT QUITE READY FOR YOU YET. FIND THE FORM YOU NEED.

Additional forms for caregivers ~, National Guard,~ and Reserve Servicemembers